includes/Classifai/Admin/SavePostHandler.php

```diff
@@ -167,6 +170,121 @@ public function classify( $post_id ) {
 		return $output;
 	}
 
+	/**
+	 * Synthesizes speech from the post title and content.
+	 *
+	 * @param int $post_id Post ID.
+	 * @return bool|int|WP_Error
+	 */
+	public function synthesize_speech( $post_id ) {
+		$normalizer   = new Normalizer();
+		$settings     = \Classifai\get_azure_text_to_speech_settings();
+		$post         = get_post( $post_id );
+		$post_content = $normalizer->normalize_content( $post->post_content, $post->post_title, $post_id );
+		$voice        = isset( $settings['voice'] ) ? $settings['voice'] : '';
```

Comment:
Can simplify this line a bit:
```suggestion
		$voice        = $settings['voice']  ?? '';
```